react-props/react-props_button/src/App.js

```diff
@@ -1,6 +1,33 @@
 import React from "react";
 import "./styles.css";
 
+function Button({ color, disabled, text, onClick }) {
+  return (
+    <button
+      disabled={disabled}
+      style={{
+        color: color,
+        height: "150px",
+        fontSize: "100px",
+        backgroundColor: "yellow",
+      }}
+      onClick={onClick}
+    >
+      {text}
+    </button>
+  );
+}
+
 export default function App() {
-  return <h1>Replace me with your component!</h1>;
+  function handleClick() {
+    console.log("I was clicked!");
+  }
+  return (
+    <Button
+      text="Hello"
```

Comment:
I have the value of text (so for you "Hello") in curly brackets, but not sure why. If this worked for you, then ignore this comment. :)